src/lambda-edge/refresh-auth/index.ts

```diff
@@ -3,69 +3,86 @@
 
 import { parse as parseQueryString, stringify as stringifyQueryString } from 'querystring';
 import { CloudFrontRequestHandler } from 'aws-lambda';
-import { getConfig, extractAndParseCookies, getCookieHeaders, httpPostWithRetry, createErrorHtml } from '../shared/shared';
-
-const { clientId, oauthScopes, cognitoAuthDomain, cookieSettings, mode, cloudFrontHeaders, clientSecret } = getConfig();
+import { getConfig, extractAndParseCookies, generateCookieHeaders, httpPostWithRetry, createErrorHtml } from '../shared/shared';
 
+let CONFIG: ReturnType<typeof getConfig>;
 
 export const handler: CloudFrontRequestHandler = async (event) => {
+    if (!CONFIG || process.env.IS_TEST_MODE) {
```

Comment:
we can remove this `process.env.IS_TEST_MODE` if setConfig() is removed.

Reply:
OK will do